src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLoadContext.cs

```diff
@@ -33,22 +34,28 @@ internal class PackageLoadContext
 
         public IVsSolutionManager SolutionManager { get; private set; }
 
+        public readonly NuGetProjectDependencyVersionLookup DependencyVersionLookup;
+
         public PackageLoadContext(
             IEnumerable<SourceRepository> sourceRepositories,
             bool isSolution,
-            INuGetUIContext uiContext)
+            INuGetUIContext uiContext,
+            NuGetProjectDependencyVersionLookup dependencyVersionLookup)
         {
             SourceRepositories = sourceRepositories;
             IsSolution = isSolution;
             PackageManager = uiContext.PackageManager;
             Projects = (uiContext.Projects ?? Enumerable.Empty<NuGetProject>()).ToArray();
             PackageManagerProviders = uiContext.PackageManagerProviders;
             SolutionManager = uiContext.SolutionManager;
+            DependencyVersionLookup = dependencyVersionLookup;
 
-            _installedPackagesTask = PackageCollection.FromProjectsAsync(Projects, CancellationToken.None);
+            _installedPackagesTask = PackageCollection.FromProjectsAsync(Projects, null, CancellationToken.None);
+            _resolvedPackagesTask = PackageCollection.FromProjectsAsync(Projects, DependencyVersionLookup, CancellationToken.None);
```

Comment:
At first I only had one package collection, but the consolidate tab relies on the installed packages collection and when I used the resolved packages for this particular tab it didn't work. That's why I decided to do two package collections, any idea how this issue could be approached?